Assets/LeapMotion/Scripts/VR/VRHeightOffset.cs

```diff
@@ -0,0 +1,39 @@
+﻿using UnityEngine;
+using UnityEngine.VR;
+using System;
+using System.Linq;
+
+public class VRHeightOffset : MonoBehaviour {
+
+  [Serializable]
+  public class DeviceHeightPair {
+    public string deviceName;
+    public float heightOffset;
+
```

Comment:
Since these are public, shouldn't they be capitalized: DeviceName and HeightOffset?